test/SetInterface_test.py

```diff
@@ -0,0 +1,63 @@
+"""Generic SetInterface tests."""
+from test.common_test import check_get_set_output
+from typing import Any
+
+import pytest
+from SetAPI.generic.SetInterfaceV1 import SetInterfaceV1
+
+STANDARD_VALUES = [None, 0, 1]
+
+
+@pytest.mark.parametrize(
+    "ws_id",
+    ["default_ws_id"],
+    indirect=["ws_id"],
+)
+@pytest.mark.parametrize(
+    "ref_args",
+    [
+        pytest.param("__SET_REF__", id="set_ref"),
+        pytest.param("__WS_NAME__SET_NAME__", id="ws_name_set_name"),
+    ],
+)
+@pytest.mark.parametrize("inc_item_info", STANDARD_VALUES)
+@pytest.mark.parametrize("inc_item_ref_paths", STANDARD_VALUES)
+@pytest.mark.parametrize("ref_path_to_set_present", STANDARD_VALUES)
+def test_get_set(
```

Comment:
This tests the core `SetInterfaceV1` module with all the different variations of inputs that can come with a `get_***_set` query. It picks a random `set` fixture (see the `random_fixture` param) and then runs the same query with different param combinations.

This test will eventually replace the tests in the individual test classes (`AssemblySet_basic_test`, `FeatureSetSet_test`, etc.) in an upcoming PR.